openwebstart/src/main/java/com/openwebstart/proxy/windows/WindowsProxyUtils.java

```diff
@@ -77,14 +77,28 @@ private static ProxyConfigurationImpl getProxyConfiguration(final String proxySe
             proxyConfiguration.setUseHttpForHttpsAndFtp(true);
             proxyConfiguration.setUseHttpForSocks(true);
 
-            final String[] split = hosts.get(0).split(Pattern.quote(":"), 2);
-            if (split.length == 1) {
-                //TODO: Default port??
-                throw new IllegalStateException("No port defined!");
-            } else {
-                //TODO: We need to check port behavior on win
-                proxyConfiguration.setHttpHost(split[0]);
-                proxyConfiguration.setHttpPort(Integer.parseInt(split[1]));
+            String proxyString = hosts.get(0);
+            try {
+                if (!proxyString.contains("://")) {
+                    proxyString = "http://" + proxyString;
```

Comment:
by prepending `http://` you implicitly set the default port to 80.
So in case the proxyString contains no protocol and no port you set the port to 80.
The protocol is also set but ignored...